modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces_{context}"]
+= Creating the default revision tag and relabeling the namespaces
+
+This procedure explains how to create the default revision tag and relabel the namespaces after you have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process using istio injection label.
+
+The `bookinfo` example application is being used for demonstration purposes with a minimal example for the `Istio` resource. For more information on configuration differences between the {SMProduct} 2 `ServiceMeshControlPlane` resource and the {SMProduct} 3 `Istio` resource, see "Configuration fields mapping between Service Mesh 2 and Service Mesh 3".
+
+.Prerequisites
+
+* You have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process by using {istio} injection label.
+
+.Procedure
+
+. Create a YAML files called `rev-tag.yml` that defines the `IstioRevisionTag` resource:
++
+.Example `IstioRevisionTag` resource
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1
+kind: IstioRevisionTag
+metadata:
+  name: default
+spec:
+  targetRef:
+    kind: IstioRevision
+    name: ossm-3-v1-24-3
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f rev-tag.yaml
+----
+
+. Verify the status of the `IstioRevisionTag` resource by running the following command:
++
+[source,terminal]
+----
+$ oc get istiorevisiontags
+----
++
+.Example output
+[source,terminal]
+----
+NNAME      STATUS                    IN USE   REVISION        AGE
+default    NotReferencedByAnything   False    ossm-3-v1-24-3  18s
+----
+
+. Add the `istio-injection=enabled` label to the `bookinfo` namespace, and remove the `istio.io/rev` label by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo istio-injection=enabled istio.io/rev-
+----
++
+[NOTE]
+====
+Repeat this step for each namespace you are migrating.
```

Comment:
Repeat steps 4 and 5. Labeling + restart

Reply:
Done. I think I like it better the way I just changed it.